circuit_knitting_toolbox/circuit_cutting/cutting_evaluation.py

```diff
@@ -94,13 +94,25 @@ def execute_experiments(
     if isinstance(circuits, dict):
         if circuits.keys() != subobservables.keys():
             raise ValueError(
-                "The keys for the circuits and observabes dicts should be equivalent."
+                "The keys for the circuits and observables dicts should be equivalent."
             )
         if isinstance(samplers, dict) and circuits.keys() != samplers.keys():
             raise ValueError(
                 "The keys for the circuits and samplers dicts should be equivalent."
             )
 
+    if isinstance(samplers, dict):
```

Comment:
Looks great. Could we add this to the docstring above?

Reply:
Done